plugins/la_spec.py

```diff
@@ -83,18 +85,6 @@ def __init__(self, name, detector, dataflow, emitter, l2=None, analyzer=None, **
             self.acquireAllPol = model.BooleanVA(False)
 
     # Copy from ARSettingsStream (to only change the axis when playing)
```

Comment:
Is this copying still done? I can't see where this comment is referring towards currently

Reply:
Most of the code below this comment is from the ARSettingsStream (note, this is *not* the SEMARMDStream, which you are familiar with).